README.md

```diff
@@ -178,3 +188,61 @@ openstack esi switch port disable access
 
 - `switch`: Switch
 - `switchport`: Switchport
+
+### `openstack esi switch port enable trunk`
+
+Attach VLAN to a switchport as a trunk on a switch.
+
+```
+openstack esi switch port enable trunk
+   <switch>
+   <switchport>
+   <vlan>
+```
+
+- `switch`: Switch
+- `switchport`: Switchport
+- `vlan`: VLAN
```

Comment:
we should document that this will be the native-vlan

Reply:
Done!